tutorials/Robust_Linear_Regression.ipynb

```diff
@@ -1129,7 +1129,7 @@
     "        \"plot_width\": 500,\n",
```

Comment:
<p>Do you mind addressing the DataConversionWarning here ;)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>